tests/test_client.py

```diff
@@ -585,11 +584,13 @@ def test_atexit(tmpdir, monkeypatch, num_messages):
     import time
     from sentry_sdk import init, transport, capture_message
 
-    def send_event(self, event):
+    def capture_envelope(self, envelope):
         time.sleep(0.1)
-        print(event["message"])
+        event = envelope.get_event() or dict()
+        message = event.get("message", "")
```

Comment:
Can `envelope.get_event()` return something falsy here? And can `message` be `None` on the event?